test/to_string_test.hpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+using namespace std;
+void toStringTest()
+{
+    int i;
+    cin>>i;
+    string  palindrom = to_string (i);
+    cout<<palindrom<<" testl"<<endl;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace

```suggestion
    cin >> i;
    string palindrom = to_string(i);
    cout << palindrom << " testl" << endl;
```